pydoover/docker/application.py

```diff
@@ -1045,6 +1045,7 @@ def run_app(
     start: bool = True,
     setup_logging: bool = True,
     log_formatter: logging.Formatter = None,
+    log_filters: logging.Filter | list[logging.Filter] = None,
```

Comment:
It'd be great to add this parameter to the docstring of `run_app` so that it appears in the pydoover docs. Ie. similar to `log_formatter` - https://github.com/getdoover/pydoover/blob/main/pydoover/docker/application.py#L1078

I was going to show you where this function is in the docs for an example before realising that I forgot to include it, oops... but as an example it would show up like the parameters [here](https://pydoover.readthedocs.io/en/latest/applications/docker.html#pydoover.docker.Application.set_tag)